frontend/zc_messaging/src/pages/message-board/index.jsx

```diff
@@ -365,14 +375,14 @@ const MessagingBoard = () => {
           <MessageWrapper>
             <MessageRoomViewHeader name={`#${roomName}`} />
             <MessageBoard
-              isLoadingMessages={isLoadingRoomMessages}
-              messages={roomChats || []}
+              isLoadingMessages={isFetching}
+              messages={isFetching ? [] : roomChats || []}
               onSendMessage={sendMessageHandler}
               onReact={reactHandler}
               onSendAttachedFile={SendAttachedFileHandler}
               currentUserId={authUser?.user_id}
               height={"92vh"}
-              onHandleScroll={handleScroll}
+              onHandleScroll={event => handleScroll(event)}
```

Comment:
I'm guessing this is very dependent on the PR that you made to zc main.

Reply:
The handle scroll function was emitting without it being called. Found a fix. so i have removed the event